pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -666,4 +681,73 @@ public boolean hasKitReceived(Connection connection, String ddpParticipantId) {
         }
         return false;
     }
+
+    public ResultSet getKitsInDatabaseByInstanceId(DDPInstance ddpInstance) {
```

Comment:
From a separation of concerns point of view, it would be better to return DTOs instead of result sets.
Is there a reason you chose this approach?